.vscode/launch.json

```diff
@@ -29,8 +29,7 @@
             "args": ["--extensionDevelopmentPath=${workspaceRoot}" ],
             "stopOnEntry": false,
             "sourceMaps": true,
-            "outFiles": ["${workspaceRoot}/out/src/**/*.js"],
-            "preLaunchTask": "cover:disable"
```

Comment:
Why did you remove this?

Reply:
some of these changes I did not make. @abist / @alanrenmsft made, and as such, I refactored and fixed the parts that needed to be changed

Reply:
Please remove it for now unless there's a reason to keep it, we can do a follow up PR if there's a reason it needs to be removed.